concent_api/core/tests/test_auth_report_computed_task.py

```diff
@@ -953,8 +953,8 @@ def test_provider_forces_computed_task_report_and_requestor_does_not_respond_sho
 
         self.assertEqual(response.status_code,                                                                  200)
         self.assertEqual(force_report_computed_task_from_view.timestamp,                                        self._parse_iso_date_to_timestamp("2017-12-01 11:00:05"))
-        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute.timestamp,   self.force_report_computed_task.report_computed_task.task_to_compute.timestamp)
-        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute,             self.force_report_computed_task.report_computed_task.task_to_compute)
+        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute.timestamp,   self.force_report_computed_task.report_computed_task.task_to_compute.timestamp)  # pylint: disable=no-member
+        self.assertEqual(force_report_computed_task_from_view.report_computed_task.task_to_compute,             self.force_report_computed_task.report_computed_task.task_to_compute)  # pylint: disable=no-member
```

Comment:
If pylint warning are one by one make them to be in one straight line

Reply:
Done.